package.json

```diff
@@ -59,8 +59,11 @@
   },
   "dependencies": {
     "@vscode/vscode-languagedetection": "1.0.15",
+    "argon2": "^0.28.2",
     "applicationinsights": "1.0.8",
     "chokidar": "3.5.1",
+    "cookie-parser": "^1.4.5",
+    "express": "^4.17.1",
```

Comment:
Is there a way to build auth on plain http without pulling express?

Reply:
Although let's go with it if it is easier. We need to make sure that it still works in budnled variant.

Do follwoing: 
- yarn gulp server
- it should product the prod ready server under `/workspace`
- then try to start it and make sure that deps are found, if not then something should be changed in `/workspace/vscode/remote` folder.

Reply:
Generally adding new deps is not welcomed.

Reply:
It's generally possible (see every commit before 61bc66c). But that relies on manually parsing cookies and request bodies, which works, but looks really hacky. I don't care, the express solution is just way cleaner.

I can remove all new dependencies if you want. We can keep it at plain passwords without the `argon2` dependency as well.

Reply:
try to build it for final distribution and run if you make it work let's go with it. We can revert later, we don't promise backward compatibility here to anyone :D

Reply:
Which file/script starts the production server? Running `server.js` under `server-pkg/out` does nothing.

Reply:
correct

Reply:
> correct

Is it supposed to do nothing? If I start it, I end up back at the console without any message.